data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -367,6 +367,15 @@ bootkube_podman_run \
 	"${CLUSTER_BOOTSTRAP_IMAGE}" \
 	start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
 
+if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+then
+	echo "Waiting for ceo to finish..."
+	bootkube_podman_run \
+		--volume /etc/kubernetes:/etc/kubernetes:z \
+		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+		/usr/bin/wait-for-ceo --kubeconfig /etc/kubernetes/kubeconfig
```

Comment:
@retroflexer This should be `/usr/bin/cluster-etcd-operator wait-for-ceo`
https://github.com/hexfusion/installer/blob/811d527e1e25eda0c26024d92db60a698e7fcf0c/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L123